releases/im-in-your-mind-fuzz/index.md

```diff
@@ -9,11 +9,11 @@ title: "I’m In Your Mind Fuzz"
 > When I’m in your mind fuzz  
 > Then I'm in your mind
 
-Release Date: 31 October 2014
+Release Date: 31 October 2014.
 
-With a clean Krautrock inspired foundation, KGATLW make their first big evolution with a pivot away from lo-fi production. im-in-your-mind-fuzz moves more confidently between driving and relaxed psychedelic sounds, and features the Flute as part of a project to learn a new instrument every year. The first album recorded in a professional studio, the first published internationally, and the first truly essential, realised, KGATLW album.
+With a clean Krautrock inspired foundation, KGATLW make their first big evolution with a pivot away from lo-fi production. I’m In Your Mind Fuzz moves more confidently between driving and relaxed psychedelic sounds, and features the Flute as part of a project to learn a new instrument every year. The first album recorded in a professional studio, the first published internationally, and the first truly essential, realised, KGATLW album.
```

Comment:
Now that I'm used to seeing "KGLW", reading "KGATLW" here strikes me as funny. Just an observation